### ISSUE TITLE
Narrator announcing incorrect info for Calendar dates

### ISSUE THREAD

**Issue by DhirajKhot** [OfficeDev/office-ui-fabric-react#8415]
![image](https://user-images.githubusercontent.com/20378231/54733711-21d6b000-4bc1-11e9-989f-3807b4b52519.png)

## Actual behavior:
The current date is announced as editable.
For other dates Narrator is announcing the column header but the dates are not announced.
It is announcing some irrelevant info - no current char.

## Expected behavior:
Current Date should not be announced as editable.
Dates should be correctly announced along with column Header. No irrelevant announcement should be made.

## Documentation describing expected behavior
[MAS 4.2.3 - Row, Column, and Headers](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B30e4fb40-dcc1-45b4-b714-5e2c28b23ae8%7D)

**Comment by khmakoto** [comment]
Hi @DhirajKhot, from what I can see you're using scan mode for the non-current dates, in which case Narrator is announcing irrelevant info and not announcing the dates. As the inherent function of scan mode is to break out of author's intent when it comes to navigation and dictation it doesn't surprise me that the dates are not being read. I tried reproing with scan mode off and the dates were being announced correctly. Could you try this and post if you could repro it?

Regarding the current date being announced as `editable`, it seems that is a dupe of #5718. They have a proposed solution there that I'll try once I drill down on some other issues. Thanks!

**Comment by DhirajKhot** [comment]
Hello @khmakoto 
Thanks for your info. I can see that the Narrator does read out well in scan mode off. 

As you said this has now only the issue of narrator reading the dates as editable. This should be fixed.

**Comment by khmakoto** [comment]
Thanks for checking! The PR above is aiming to fix the dates being read as editable.
